src/vm-memory/src/mmap.rs

```diff
@@ -41,6 +43,9 @@ pub use vm_memory_upstream::mmap::{check_file_offset, Error};
 // The maximum number of bytes that can be read/written at a time.
 static MAX_ACCESS_CHUNK: usize = 4096;
 
+// The number of guard pages per region is a multiple of 2.
+const GUARD_NUMBER: usize = 2;
```

Comment:
I would name this to something like `GUARD_PAGE_COUNT` that is in line with what this constant does. `GUARD_NUMBER` may be misleading IMO.